apps/meteor/app/api/server/v1/users.ts

```diff
@@ -114,18 +115,24 @@ API.v1.addRoute(
 			if (userData.name && !validateNameChars(userData.name)) {
 				return API.v1.failure('Name contains invalid characters');
 			}
+			const auditStore = new UserChangedAuditStore({
+				_id: this.bodyParams.userId,
```

Comment:
Is this correct? I mean, shouldn't it be the logged user id instead? Otherwise the `actor` will mix the information from the users with the _id of the user that was edited and the username of the use who edited it.

```
 "actor" : {
        "_id" : "rocket.cat",
        "IP" : "...",
        "useragent" : "...",
        "username" : "marcos.defendi",
        "type" : "user"
    },
```